specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/examples/VirtualNetworkGatewayConnectionGetIkeSas.json

```diff
@@ -0,0 +1,14 @@
+{
+    "parameters": {
+        "api-version": "2020-08-01",
+        "subscriptionId": "subid",
+        "resourceGroupName": "rg1",
+        "virtualNetworkGatewayConnectionName": "vpngwcn1"
+    },
+    "responses": {
+        "202": {},
+        "200": {
+            "body": "\"{\"Status\":\"Successful\",\"Data\":null}\""
```

Comment:
Is the response correct? The API is supposed to return an array of MainModeSaPropertiesFormat. Can you please add a real example? Also it doesn't need to be escaped.

Reply:
The POST operation returns a URL in 'Azure-AsyncOperation' in the headers using which a GET operation needs to be executed to get the IKE SAs